scripts/patch-realsense-ubuntu-odroid-xu4-4.14.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+
+#This script is almost an exact copy of the patch-realsense-ubuntu-lts.sh script, tested on 4.14.47-132.
+#There only two small additions / checks. Actually, this script also works for normal ubuntu-lts
+
+#Break execution on any error received
+set -e
+
+#Locally suppress stderr to avoid raising not relevant messages
+exec 3>&2
+exec 2> /dev/null
+con_dev=$(ls /dev/video* | wc -l)
+exec 2>&3
+
+if [ $con_dev -ne 0 ];
+then
+	echo -e "\e[32m"
+	read -p "Remove all RealSense cameras attached. Hit any key when ready"
+	echo -e "\e[0m"
+fi
+
+#Include usability functions
+source ./scripts/patch-utils.sh
+
+# Get the required tools and headers to build the kernel
+#sudo apt-get install linux-headers-generic build-essential git
+sudo apt-get install build-essential git
+
+#Packages to build the patched modules
+require_package libusb-1.0-0-dev
+require_package libssl-dev
+
+retpoline_retrofit=0
+
+LINUX_BRANCH=$(uname -r)
+PLATFORM=$(uname -n)
+if [ "$PLATFORM" = "odroid" ]; then
```

Comment:
Its just working if the pc name as default "odroid"
In any other case the function `choose_kernel_branch` returns the error with unsupported kernel version.